torch/csrc/lazy/ts_backend/tensor_aten_ops.cpp

```diff
@@ -105,14 +105,12 @@ torch::lazy::LazyTensorPtr expand(
     const torch::lazy::LazyTensorPtr& input,
     std::vector<int64_t> size) {
   auto input_shape = input->shape();
-  auto output = torch::lazy::LazyTensor::Create(
+  return torch::lazy::LazyTensor::Create(
```

Comment:
The TS backend uses functionalization (so top-level tensors running with the LTC-TS backend will show up as "FunctionalTensorWrappers" with storages), but that's not technically true for XLA yet (until I can land my patch). Given that, is removing this safe? If this doesn't break anything on the XLA side, then this seems ok to me.

Reply:
Yea, the XLA side hasn't used the `lazy::Tensor` infra yet. So, it should be safe.